core/src/main/resources/reference.conf

```diff
@@ -27,7 +27,22 @@ avastRabbitMQConnectionDefaults {
 
   networkRecovery {
     enabled = true
+
+    type = "linear" // exponential, linear
+
+    // common values:
+    initialDelay = 1s
+
+
+    // "linear" values:
     period = 5s
+
+
+    // "exponential" values:
+    period = 5s
```

Comment:
Shouldn't these be commented out when they are not used. I'd prefer that.

Reply:
It's meant like defaults so you can just switch to `exponential` without specifying those values.

Reply:
You're right. Reworked.